data-esnext.js

```diff
@@ -305,8 +305,8 @@ exports.tests = [
       return typeof globalThis === 'object' && globalThis && globalThis === actualGlobal && !globalThis.lacksGlobalThis && globalThis.__system_global_test__ === 42;
     */},
     res: {
-      babel6: false,
```

Comment:
we should keep these; they represent verified results

Reply:
Ok.